client/js/store/map/ProcedureMapSettings.js

```diff
@@ -111,8 +111,12 @@ export default {
             }
 
             commit('setItem', { key: 'procedureMapSettings', value: procedureMapSettings })
+
             return procedureMapSettings
           })
+          .catch(() => {
+            dplan.notify.error(Translator.trans('error.api.generic'))
+          })
```

Comment:
I think it does not catch. I could reproduce the map not showing, but it didn't show any error messages and the breakpoint did not stop (instead it went into the response part).

Reply:
Then You got a different error. That might be. But since I can't reproduce any, I try to minimize the effects of everything that could happen. And I have been reported, that the request fails "sometimes"